### DISCUSSION TITLE
Ability To Pin Certain Process Settings

### DISCUSSION THREAD

**Discussion by Flumeango** [OrcaSlicer/OrcaSlicer#7411] [Ideas]
It would be cool :)
Mostly for options that you change on the regular like ironing, support, seam position, etc.

**Comment by LJKraus** [comment]
To some extent, this already exists.

 Make your changes, rename and save the Process as a User preset.  

Next time you use OrcaSlicer it will open using the last-used Process. If that is not the process you want, the dropdown list of presets will at least start within the list of other User presets.
